### DISCUSSION TITLE
Card Size and Location coordinates

### DISCUSSION THREAD

**Discussion by knox1138** [home-assistant/frontend#13209] [Dashboard]
I was directed here for feature requests, so my request is to be able to arrange cards by giving them an x and y coordinate, and to define card size with x and y dimensions. It just seems more logical.

**Comment by ildar170975** [comment]
What happens if some of your cards is an Entity card and you added some entities? Will the height of the card change? Will coords of other cards change? Or a user change calculate these values by himself and then change them for other cards?

**Comment by knox1138** [comment]
If you added entities to a card you would change the card dimensions to
suit your needs. I would think nothing about your card size would change
unless you changed it, regardless of adding or removing entities.

**Comment by KTibow** [comment]
Hardcoding things is bad, especially when you use pixels. It means if the dimensions change, either stuff gets resized, cropped, or moved. If you want something like this, you could find another dashboard, like [HADashboard](https://appdaemon.readthedocs.io/en/latest/DASHBOARD_INSTALL.html).

**Comment by knox1138** [comment]
I prefer that level of control, and think giving users less options is bad.

**Reply by KTibow** [reply]
You already have options. If you need more control, you can install layout-card.

**Comment by knox1138** [comment]
Layout card doesn't allow me to define the size or location of the card
exactly as I desire, it just gives me better options than not much.

**Reply by KTibow** [reply]
What are you looking for? Also you know you can reply instead of making an answer, right...?

**Comment by knox1138** [comment]
What am I looking for?  Complete control over the exact location and size
of cards. Also, I have no idea what you mean by reply instead of making an
answer. I'm getting emails and hitting reply. Is there another place where
my replies are not coming across as replies?

**Reply by KTibow** [reply]
See https://github.com/home-assistant/frontend/discussions/13209.

**Reply by knox1138** [reply]
Oh, well this format looks completely different and what you said makes sense.
